<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="3" parallel="tests" name="Failed suite [Suite-Collection]" guice-stage="DEVELOPMENT">
  <test thread-count="3" parallel="tests" name="DataProviderWithExcel_003(failed)">
    <classes>
      <class name="test.DataProviderWithExcel_003">
        <methods>
          <include name="afterMethod"/>
          <include name="beforeMethod"/>
          <include name="f" invocation-numbers="0"/>
        </methods>
      </class> <!-- test.DataProviderWithExcel_003 -->
    </classes>
  </test> <!-- DataProviderWithExcel_003(failed) -->
  <test thread-count="3" parallel="tests" name="DataProviderWithExcel_002(failed)">
    <classes>
      <class name="test.DataProviderWithExcel_002">
        <methods>
          <include name="afterMethod"/>
          <include name="f" invocation-numbers="0"/>
          <include name="beforeMethod"/>
        </methods>
      </class> <!-- test.DataProviderWithExcel_002 -->
    </classes>
  </test> <!-- DataProviderWithExcel_002(failed) -->
</suite> <!-- Failed suite [Suite-Collection] -->
